cmd/swarm/main.go

```diff
@@ -155,7 +155,7 @@ var (
 	}
 	SwarmUploadMimeType = cli.StringFlag{
 		Name:  "mime",
-		Usage: "force mime type",
+		Usage: "force mime type on upload",
```

Comment:
actually I'm still not sure what this means.

Reply:
forces the mime type in the manifest...?

Reply:
what does that mean? force the inclusion of mime type information into the manifest? no. that cannot be because there is always mime type info in the manifest.

forces some form of auto detection of mime type? by file extension maybe? ... and what happens if we don't 'force'? 
I don't get it.

Reply:
This flag sets an _explicit_ value for the mime type that ultimately becomes `ContentType` in the resulting manifest.

If this flag is not set, then we auto-detect the mime type from the file.

For example, `swarm up file.json` would lead to a mime type of `application/json` (i.e. auto-detected), but you can "force" the mime type with `swarm --mime text/plain up file.json` if you want to.

Reply:
ah! That makes a lot more sense.
"force mime type on upload" --> "manually specify MIME type" 
?